src/cubit_utils.py

```diff
@@ -0,0 +1,294 @@
+import cubit
+from pathlib import Path
+import os
+import inspect
+import subprocess
+
+
+def init_cubit(cubit_flag):
+    """Initializes Coreform Cubit with the DAGMC plugin.
+
+    Arguments:
+        cubit_flag (bool): flag to indicate whether Coreform Cubit has been
+            initialized.
+
+    Returns:
+        cubit_flag (bool): flag to indicate whether Coreform Cubit has been
+            initialized.
+    """
+    if not cubit_flag:
+        cubit_plugin_dir = (
+            Path(os.path.dirname(inspect.getfile(cubit))) / Path('plugins')
+        )
+        cubit.init([
+            'cubit',
+            '-nojournal',
+            '-nographics',
+            '-information', 'off',
+            '-warning', 'off',
+            '-commandplugindir',
+            str(cubit_plugin_dir)
+        ])
+        cubit_flag = True
+
+    return cubit_flag
+
+
+def import_step_cubit(filename, import_dir):
```

Comment:
should we call `init_cubit` in each of these functions?  After the first time, it will basically do nothing, but at least we'll always know it's been initialized.

Reply:
For the Cubit utility script, I only included `init_cubit()` in `import_step_cubit()` since, presumably, if the export functions are being called, Cubit has already been initialized.

However, `init_cubit()` is included in all functions with Cubit functionality outside `cubit_utils.py` that don't rely on Cubit operations already happening.